requirements-base.txt

```diff
@@ -0,0 +1,30 @@
+gunicorn>=19.9.0,<19.9.99
+
+# Not sure if Django 2 will work yet, hold back to LTS
+Django>=1.11,<1.11.99
+whitenoise>=3.3.1,<3.3.99
+
+django-nvd3>=0.9.7,<0.9.99
```

Comment:
we might drop this too. I am no longer using nvd3, just d3 directly. @pushyamig how about you?